src/migrations/20220802000000-goal-objective-metadata-caching.js

```diff
@@ -0,0 +1,380 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+      try {
+        const loggedUser = '0';
+        // const transactionId = '';
+        const sessionSig = __filename;
+        const auditDescriptor = 'RUN MIGRATIONS';
+        await queryInterface.sequelize.query(
+          `SELECT
+            set_config('audit.loggedUser', '${loggedUser}', TRUE) as "loggedUser",
+            set_config('audit.transactionId', NULL, TRUE) as "transactionId",
+            set_config('audit.sessionSig', '${sessionSig}', TRUE) as "sessionSig",
+            set_config('audit.auditDescriptor', '${auditDescriptor}', TRUE) as "auditDescriptor";`,
+          { transaction },
+        );
+      } catch (err) {
+        console.error(err); // eslint-disable-line no-console
+        throw (err);
+      }
+
+      // Disable logging while doing mass updates
+      try {
+        await queryInterface.sequelize.query(
+          `
+          SELECT "ZAFSetTriggerState"(null, null, null, 'DISABLE');
+          `,
+          { transaction },
+        );
+      } catch (err) {
+        console.error(err); // eslint-disable-line no-console
+        throw (err);
+      }
+
+      try {
+        await queryInterface.addColumn('ActivityReportGoals', 'name', {
+          type: Sequelize.TEXT,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('ActivityReportGoals', 'status', {
+          type: Sequelize.STRING,
+          allowNull: true,
+        }, { transaction });
+
+        await queryInterface.addColumn('ActivityReportGoals', 'timeframe', {
+          type: Sequelize.STRING,
```

Comment:
```suggestion
          type: Sequelize.TEXT,
```

Turns out I also misidentified the offending column , tested it this time. BTW, opened [this PR ](https://github.com/HHS/Head-Start-TTADP/pull/1016) off this branch to get the plumbing straightened away